prod/wallet-options-v4.json

```diff
@@ -298,56 +298,6 @@
           "iSignThisDomain": "https://verify.isignthis.com"
         }
       },
-      "sfox": {
-        "countries": [
-          "US"
-        ],
-        "states": [
-          "AK",
-          "AZ",
-          "CA",
-          "CO",
-          "DE",
-          "ID",
-          "IL",
-          "IN",
-          "KY",
-          "LA",
-          "MA",
-          "MD",
-          "ME",
-          "MI",
-          "MO",
-          "MT",
-          "NE",
-          "NH",
-          "NJ",
-          "NV",
-          "OK",
-          "OH",
-          "PA",
-          "RI",
-          "TN",
-          "TX",
-          "UT",
-          "VA",
-          "WI",
-          "WV",
-          "WY"
-        ],
-        "config": {
-          "production": true,
-          "apiKey": "f31614a7-5074-49f2-8c2a-bfb8e55de2bd",
```

Comment:
@plondon confirming we dont need this API key to retrieve historical SFOX orders that we may still show in the UI

Reply:
Good catch, seems we do.

https://github.com/blockchain/bitcoin-sfox-client/blob/master/src/api.js#L54